SessionMessagingKit/Open Groups/OpenGroupAPI.swift

```diff
@@ -192,9 +193,10 @@ public enum OpenGroupAPI {
     /// could return: `{"capabilities": ["sogs", "batch"], "missing": ["magic"]}`
     public static func preparedCapabilities(
         authMethod: AuthenticationMethod,
-        using dependencies: Dependencies
+        using dependencies: Dependencies,
+        skipAuthentication: Bool = false
```

Comment:
`using dependencies` should always be the last parameter - please put `skipAuthentication` before it